interface/weno/jsonScript.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
+ /*
+ * The purpose of this is to on the fly output the json code needed to transmit the prescriptions individually
+ */
+
+$fake_register_globals=false;
+
+$sanitize_all_escapes=true;	
+
+include_once("../globals.php");
+include_once($srcdir."/patient.inc");
+include_once("transmitDataClass.php"); //class include file, breaking away from procedure coding
+
+$date = date("Y-m-d");
+$pid = $GLOBALS['pid'];
+$uid = $_SESSION['authUserID'];
+
+$i = rand();
+$fillData = filter_input(INPUT_GET, "getJson");
+
+
+$fill = explode(",", $fillData);
+
+$id = $fill[0]; //setting the pharmacy ID for later
+
+array_shift($fill);  //removing the pharmacy from the array
+
+//created a loop in case
+foreach($fill as $data){
+$pInfo = getPatientData($pid);
+
+
+$pfname     = $pInfo['fname'];
+$plname     = $pInfo['lname'];
+$pstreet     = $pInfo['street'];
+$ppostalCode = $pInfo['postal_code'];
+$pcity       = $pInfo['city'];
+$DOB        = $pInfo['DOB'];
+$sex        = $pInfo['sex'];
+
+function fillJsonPatient($pfname,$plname,$pstreet,$ppostalCode,$pcity,$DOB,$sex){
+
+    if($sex == "Male"){ $sex = "M";}
+	if($sex == "Female"){$sex = "F";} //has to be a single letter
+	
+$patient ='[{"patient": {"lname" : "'.$plname.'","fname" : "'.$pfname.'","street" : "'.$pstreet.'","city" : "'.$pcity.'","postal" : "'.$ppostalCode.'","DOB" : "'.$DOB.'","Sex" : "'.$sex.'"}},';
```

Comment:
Safety was not a concern while trying to figure out how to build a very specific JSON array from all the gathered information then return that script to the page to be transmitted.  The data has to be in the order that it is configured. I couldn't see how to pull the data in the order that I have it with all of the headers and then convert it to a JSON format. I let the code developed as I figure things out. I did request input from you Brady but I understand you are bombarded day in and day out with development questions and you can't get to everyone. So, I did the best I could with what I know how to do and could find. 

Which values need escaping?